### ISSUE TITLE
"Failed to resolve Reference" when there are includes

### ISSUE THREAD

**Issue by dhaarbrink** [thephpleague/openapi-psr7-validator#127]
I have an `openapi: 3.0.1` specification. It has several paths and all definitions are in separate files, next to the main file.

This is an example of a path:

     /orders/{orderId}:
      get:
        tags:
          - orders
        summary: Get an order by order id
        description: Get an order by order id
        parameters:
          - name: orderId
            in: path
            required: true
            schema:
              type: string
          - in: query
            name: include
            required: false
            schema:
              $ref: "./includes.yaml#/components/schemas/OrderIncludes"
        responses:
          200:
            description: The requested order
            content:
              '*/*':
                schema:
                  $ref: 'responses.yaml#/components/schemas/Order'

When I try to get a validator:

        $validator = (new ValidatorBuilder())
            ->fromYaml(file_get_contents($this->swaggerFile))
            ->getServerRequestValidator();

I get the following error:

> Failed to resolve Reference './includes.yaml#/components/schemas/OrderIncludes' to cebe\\openapi\\spec\\Schema Object: Failed to evaluate pointer '/components/schemas/OrderIncludes'. Array has no member OrderIncludes at path '/components/schemas'.

My includes.yaml looks like this (relevant schema only):

    openapi: 3.0.1
    info:
      title: Includes definitions
      version: 1.0.0
      description: Includes definitions
    paths: {}
    
    components:
      schemas:
        OrderIncludes:
          type: string
          enum:
            - orderItems
            - customer
            - payments
            - promotion

I have tried to specify the file as:
- includes.yaml
- ./includes.yaml
- /var/www/html/includes.yaml

But I always get the same error. What I gather from the error message is that it can't find the requested schema in the file, but it is not complaining about finding the file itself.

Where am I going wrong?

**Comment by cebe** [comment]
> But I always get the same error. What I gather from the error message is that it can't find the requested schema in the file, but it is not complaining about finding the file itself.

correct, the file seems to be fine, just finding the data inside of the file fails. looks like a bug to me.

**Comment by dhaarbrink** [comment]
I have version 1.5.2 installed, installed it just today
